components/support/nimbus-fml/src/client/inspector.rs

```diff
@@ -122,6 +113,20 @@ impl FmlFeatureInspector {
     }
 }
 
+fn syntax_error(
+    message: &str,
+    line: usize,
+    col: usize,
+    _highlight: &str,
```

Comment:
This is unused for now. In later PRs, this will be used to generate a `CursorSpan`.